### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by R-Wang97**
Implemented a rough login screen.
Endpoints have not been defined yet, and it hasn't been styling.

### CODE REVIEW

File: src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+					<form action="/" method="post">
+						<div className="form-group">
+			              <input name="username" className="form-control" placeholder="Username" required/>
+			            </div>
+			            <div className="form-group">
+			              <input name="password" type="password" className="form-control" placeholder="Password" required/>
+			            </div>
+			            <button type="submit" className="btn btn-primary center-block">Login</button>
+					</form>
+					<h5>Don't have an account? Sign up <a href="">here</a>.</h5>
```
Comment (DavidLu1997) [inline src/components/Login.js]: Do we need to escape the `'`?

**Comment by DavidLu1997** [comment]
+1

File: src/components/Login.js (hunk 1 above)
Comment (Will-Lo) [inline src/components/Login.js]: You don't need to because it's in a html tag

**Comment by Will-Lo** [comment]
+1

**Comment by R-Wang97** [comment]
Merging

**Pull request merged**